src/Root/routes.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Route, IndexRoute } from 'react-router';
+import AuthContainer from '../Auth/AuthContainer';
+import LandingPage from './LandingPage';
+
+export default (
+  <Route path="/" component={AuthContainer}>
+    <IndexRoute component={LandingPage} />
```

Comment:
Do you want to consider using `` [Redirect](https://github.com/reactjs/react-router/blob/master/docs/API.md#redirect) `` instead of `` IndexRoute ``?